app/models/manageiq/providers/openstack/cloud_manager/cloud_volume.rb

```diff
@@ -51,6 +56,27 @@ def raw_delete_volume
     raise MiqException::MiqVolumeDeleteError, e.to_s, e.backtrace
   end
 
+  def cloud_volume_backup_create(options)
```

Comment:
@durandom I think it's a bad idea; what if we try and add support for backups of a swift store or something?  Shouldn't we be able to distinguish the two?  Or do you think it doesn't matter?